### PR TITLE
Added continents, weekly file creation

### PR DESCRIPTION
**Pull request by daveluo**
- added new column assigning all owid_location to continents, based on this source: https://ourworldindata.org/world-region-map-definitions
- fixed some sparseness in the collect_weekstartdate and collect_yearweek columns across the timeseries
- added weekly aggregate file creation with an optional flag "--daily-only" set to False by default
- pre-calculated some convenience columns in the weekly file: weekly cases per mil and sequences over new cases

### CODE REVIEW

File: scripts/merge-gisaid-owid.py [hunk 1]
```diff
@@ -201,7 +204,8 @@ def pivot_merged_df(merged_df):
 
     # merge in owid cases columns which are date-dependent
     cols = ['owid_location', 'owid_date', 'owid_new_cases', 'owid_new_cases_smoothed',
-            'collect_yearweek', 'collect_weekstartdate']
+            # 'collect_yearweek', 'collect_weekstartdate'
```
Comment (julia326) [inline scripts/merge-gisaid-owid.py:207]: could probably just remove these?

File: scripts/merge-gisaid-owid.py [hunk 2]
```diff
@@ -31,6 +31,9 @@
 parser.add_argument('--gisaid-metadata-file', default='', help='GISAID metadata file export path')
 parser.add_argument('--merged-gisaid-owid-out', default='',
     help='Path where to write merged GISAID/OWID CSV')
+parser.add_argument('--daily-only', action='store_true', default=False,
```
Comment (julia326) [inline scripts/merge-gisaid-owid.py:34]: just so we're not confused later, may be worth saying that the weekly file will be created regardless as part of the help string

**Review by julia326 — APPROVED** [review]
thanks, this looks great! feel free to merge after 2 small comments

File: scripts/merge-gisaid-owid.py (hunk 2 above)
Comment (daveluo) [reply scripts/merge-gisaid-owid.py]: switched this for clarity to a -make-weekly-file flag that's True if present and False if omitted

File: scripts/merge-gisaid-owid.py (hunk 1 above)
Comment (daveluo) [reply scripts/merge-gisaid-owid.py]: done!

**Pull request merged**